src/odemis/driver/smaract.py

```diff
@@ -900,6 +900,11 @@ class SA_MC_Event(Structure):
         ("u", SA_MC_EventData),
         ]
 
+    def __str__(self):
+        return "SA_MC_EVenet {type: %s, i32: %s}" % \
```

Comment:
I'm not sure why you need it to be pretty, but if you want to have such a function, please spell correctly the class name, otherwise it's confusing: SA_MC_EVenet -> SA_MC_Event

Reply:
Just to make the logs nicer.